src/usr/pnor/sfc_ibm.C

```diff
@@ -638,19 +628,30 @@ errlHndl_t SfcIBM::hwInit( )
         }
         if( l_err ) { break; }
 
-#if 0   //@fixme-RTC:109860
```

Comment:
Neat.  Someone outside resolved an RTC issue without having access to RTC.

@dcrowell77 - Can you have someone double check that everything in this RTC issue is actually resolved by this commit sequence?